index.html

```diff
@@ -645,7 +646,7 @@ <h2>Status</h2>
     "ageOver": 21
   },
   <span class="highlight">"credentialStatus": {
-    "id": "https://dmv.example.gov/status/24,
+    "schemeID": "https://dmv.example.gov/status/24,
```

Comment:
This breaks the standard JSON-LD data model. We have to keep "id".

Reply:
The problem is an inconsistency between the text and the JSON example. The text refers to a status scheme but the original JSON does not contain any mention of a status scheme. Closer alignment of the text and the JSON is needed so that the reader can better understand how the JSON is an example of the text.